core/jobs_test.py

```diff
@@ -808,6 +808,8 @@ def test_raise_error_with_get_continuous_computation_class(self):
             (
                 jobs.BaseMapReduceJobManagerForContinuousComputations.  # pylint: disable=protected-access
                 _get_continuous_computation_class()
+                # Access of private attribute OK
```

Comment:
This sentence reads somewhat "abbreviated-ly" and is missing some words. Please use complete English sentences, similar to those that you would use in a prose essay.

Reply:
Also, I think the reason is slightly off. In this case, the access of the private attribute is OK, but that's because what you're really testing is that the base class raises a NotImplementedError for a specific method, and there's no other way to call that method. So yes, it's a structural test, but you should probably give more of the above detail so that it's clear that this isn't a generic thing.